LEGO1/mxtransitionmanager.h

```diff
@@ -15,6 +16,43 @@ class MxTransitionManager : public MxCore
   __declspec(dllexport) void SetWaitIndicator(MxVideoPresenter *videoPresenter);
 
   virtual MxResult Tickle(); // vtable+0x8
+
+  // OFFSET: LEGO1 0x1004b950
+  inline virtual const char *ClassName() const override // vtable+0x0c
+  {
+    return "MxTransitionManager";
+  }
+
+  // OFFSET: LEGO1 0x1004b960
+  inline virtual MxBool IsA(const char *name) const override // vtable+0x10
+  {
+    return !strcmp(name, MxTransitionManager::ClassName()) || MxCore::IsA(name);
+  }
+
+  virtual MxResult GetDDrawSurfaceFromVideoManager(); // vtable+0x14
+
+  enum TransitionType {
+    NOT_TRANSITIONING,
+    NO_ANIMATION,
+    DISSOLVE,
+    PIXELATION,
+    SCREEN_WIPE,
+    WINDOWS,
+    BROKEN // Unknown what this is supposed to be, it locks the game up
+  };
+
+  MxResult StartTransition(TransitionType p_animationType, MxS32 p_speed, undefined p_unk, MxBool p_playMusicInAnim);
+
+private:
+  undefined m_pad00[0x20];
+  MxU8 m_unk20;
```

Comment:
```suggestion
  undefined m_unk20;
```

Probably should be `undefined` as well? We could also merge the paddings.